app/src/main/cpp/HandModels.cpp

```diff
@@ -290,6 +298,9 @@ HandModels::InitializeGL() {
   m.uPerspective = vrb::GetUniformLocation(m.program, "u_perspective");
   m.uView = vrb::GetUniformLocation(m.program, "u_view");
   m.uModel = vrb::GetUniformLocation(m.program, "u_model");
+
+  // Since this is called lazily at render time, a flush may be needed with some drivers
+  glFlush();
```

Comment:
Why do we need this? AFAIK we're using double buffering so this in practice should make no differentce, shouldn't it?

Reply:
Note that now we are creating the GL program and the rendering buffers, filling in the buffers, and drawing; all in succession on a single frame.

While OpenGL spec should guarantee memory coherence after each command, there exists buggy implementations that do not have these guarantees (I have seen at least one such case). That's why I added `glFlush()` call, to be on the safe side, since it is harmless anyway (won't do anything if no explicitly incoherent memory commands have been issued).

That said, it is true that at least on Quest2 these calls are not needed. I can remove them and re-introduce them in the future if needed (maybe as a `glMemoryBarrier()` call instead).